packages/xstate-fsm/src/types.ts

```diff
@@ -11,6 +14,44 @@ export interface EventObject {
 
 export type InitEvent = { type: 'xstate.init' };
 
+export type AnyMachine = StateMachine.Machine<any, any, any>;
```

Comment:
I've added a bunch of helpers here that we already have in `xstate` itself. I wasn't sure if I should add those in the `StateMachine` namespace or if they should just live at the top of the module (like in XState)

Reply:
I've now spotted that there is already `StateMachine.AnyState` - so I probably should move most of those types to this namespace. Thoughts?

Reply:
For consistency, might be a good idea 👍

Reply:
I've moved `AnyMachine` and `AnyService`. Any thoughts about `*From` helpers? For now, I've left them at the top level. I think this is an acceptable "difference" and would match the XState exports shape more closely.